besca/pl/__init__.py

```diff
@@ -1,6 +1,6 @@
 from ._filter_threshold_plots import kp_genes, kp_counts, kp_cells, max_counts, max_genes, max_mito
 from ._split_gene_expression import gene_expr_split, gene_expr_split_stacked
-from ._celltype_quantification import celllabel_quant_boxplot, celllabel_quant_stackedbar, louvain_quant_stackedbar
```

Comment:
Do we potentially want to simply rename this to quant_stackedbar because its also still possible to quantify louvain with it (or another obs) and then this might become more obvious? Also is there any redundancy with celllabl_quant_stackedbar?

Reply:
It is completely redundant. I am fine to modify this. I will even vote to remove it for maintenance reason.

Reply:
54de25b